unity-renderer/ProjectSettings/TagManager.asset

```diff
@@ -34,7 +34,7 @@ TagManager:
   - ViewportCullingIgnored
   - FX
   - Skybox
-  - 
+  - CollisionsWithoutPointerEvent
```

Comment:
This layer is needed when we have Collisions and there is no pointer event blocker. This functionality never worked since it was never implemented, we need this layer to take into account this case

Reply:
What do you mean? Having physical collision without pointer block is currently working fine and it is using `CharacterOnly` layer

Reply:
If I remember correctly, this wasn't working, and changing the layer without knowing the exact usage of `CharacterOnly ` layer can leads to bugs, also the characlyOnly layer doesn't seem like a very accurate description

Reply:
There is a test scene here: https://sdk-test-scenes.decentraland.io/?position=0%2C20&realm=v1%7Esdk-test-scenes.decentraland.io
with a chest and a couple of walls.
there is a wall there that has physical collider and do not block the pointer event